src/unkdz.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
```

Comment:
problem here is that we have different distro's out there and the time where this code was written python2 was default - now it is python3 on all modern ones but we cannot be sure. the code has been adapted to python3 a while ago so we can force python3 here imo